lib/conditionals.md

```diff
@@ -5,8 +5,12 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What is a conditional?
 
-Replace me with your response...
+Conditionals are how a progrom decides what to do when it runs, based on a certain condition.
+For example, in an if statement, the computer will decide if a statement is true or false. If the statement is true, the code will run.
+If the statement is false, the program can continue to run by adding an else statement to the code. That way, the program can do something else if the statement is false.
```

Comment:
What would happen if a program did some stuff, had a single `if` conditional statement, and then did some more stuff? For example:

```ruby
player = "Alex"
player.points = 5
player.bonus = false

if player.bonus == true
  player.points += 5
  # increments the score up by 5
end

player.weapon = "herd of well-trained cats"
```
Would the final part of the program still run if the conditional evaluates to `false`, as it does in this example?